R/utils-code_dependency.R

```diff
@@ -313,6 +313,10 @@ get_code_dependency <- function(code, names) {
   checkmate::assert_multi_class(code, classes = c("character", "expression"))
   checkmate::assert_character(names)
 
+  if (identical(code, character(0)) || identical(code, "")) {
```

Comment:
Small suggestion to treat string that only has whitespace as an empty string

```suggestion
  if (identical(code, character(0)) || identical(trimws(code), "")) {
```